Form/View/ExportViewType.php

```diff
@@ -117,9 +138,32 @@ public function getBlockPrefix()
      */
     public function generateResponse(View $view, Request $request)
     {
-        $response = new Response();
         $parameters = $this->getParameters($view, $this->formFactory, $request);
-        
+
+        if (isset($view->getOptions()['export_type']) or $view->getOptions()['export_type'] === 'dompdf') {
```

Comment:
Preference for || instead of 'or' (convention?), as there is no precedence issue here